README.md

```diff
@@ -288,7 +292,8 @@ If you've deployed the application with `kubectl apply -f [...]`, you can
 run `kubectl delete -f [...]` with the same argument to clean up the deployed
 resources.
 
-## Conferences featuring Hipster Shop
+## Conferences Featuring Hipster Shop
+<p style="color:red">Do we need to keep this section?</p>
```

Comment:
I think so. It's a means for people to preview what the demo actually is and it's history.